sfa_dash/templates/banner.html

```diff
@@ -1,5 +1,5 @@
 <div class="container">
-  <p>This is a pre-Alpha release of the Solar Forecast Arbiter dashboard. If you encounter any issues please send us a report using our
+  <p>This is the Alpha release of the Solar Forecast Arbiter dashboard. If you encounter any issues please send us a report using our
```

Comment:
can we use versioneer to make this automatic? I imagine we'll have multiple alpha releases before the next milestone-mandated beta release.

Reply:
Sure can!